### DISCUSSION TITLE
v1.5.0 Boot Chime Upgrade Release!

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T480#8] [Announcements]
![repository-t480-template](https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/assets/72415505/5ee730f6-6717-4bff-a9e8-e8a1d95b7a7d)

# New Boot Chime Installed! 💻

This release includes the modern 2020+ MacBook boot chime and 3 other EFI folders. All of which are pretty self-explanatory and are equipped with the ```MacBookPro15,2``` or 2018 MacBook Pro SMBIOS.

**Full Changelog**: https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/compare/v1.4.1...v1.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/releases/tag/v1.5.0'>v1.5.0 Boot Chime Upgrade Release!</a>.</em>

**Comment by VinylNerd** [comment]
What are benchmarks like? I hit these numbers the other day on 3 tests in a row, tried changing my video card pre-allocated RAM in bios from 256-512, bad ideal. Also tried turning wake by thunderbolt off and i haven't been able to get scores this good again, but I'm close

There is definitely some optimization that can be done on this build

https://browser.geekbench.com/v6/compute/1929188

https://browser.geekbench.com/v6/compute/1929188

https://browser.geekbench.com/v6/cpu/5369739

**Reply by MultimediaLucario** [reply]
The EFI was originally compiled to be more optimized for the i5-8**3**50U. I am currently working on a separate version that is more optimiized for your configuration.

**Reply by VinylNerd** [reply]
what are your benchmark scores currently? below are what i get today on my build.

I notice you are using ThinkPad Assistant and YogaSMC, is there a reason for this?  YogaSMC should do everything ThinkPad Assistant can. on the other hand only real thing YogaSMC does for me is allow me to manually control fan. I am fine with default fan management. I don't find setting to performance or power savings changed anything, so i am realistically ok with using no YogaSMC and just ThinkPad Assistant, unless i am missing something. Perhaps with a 1vyrain bios unlock and CFG unlock its possible..

I am using YogaSMC but not ThinkPad Assistant

is thunderbolt working on your build? mine was working but now its not, i think i can fix it though.

lastly i would suggest an F13 -> Print-screen keyboard function :)

https://browser.geekbench.com/v6/compute/1968315

https://browser.geekbench.com/v6/compute/1968243

https://browser.geekbench.com/v6/cpu/5477191

![Screenshot 2024-03-26 at 12 14 44 PM](https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/assets/72950020/555622f8-5a6f-4ac0-bc7d-2839d3abe127)

**Reply by MultimediaLucario** [reply]
The geek bench score were in the 2000s range. Regardless of that the scores don't do the performance justice.

**Reply by VinylNerd** [reply]
> The geek bench score were in the 2000s range. Regardless of that the scores don't do the performance justice.

What would be better to do the performance justice?

Its crucial to test for performance differences when posting public efis. Saying geekbench results dont do performance justice is not true, its important, and valnoxy missed this step when he released monterey and then performance was shit before OCLP patch for HD4600

You are using wrong platform ID for gfx - see below

8250u-8650u are Kaby Lake R

![Screenshot 2024-03-27 at 1 02 26 PM](https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/assets/72950020/243bcfb0-2ad1-4e4a-b126-7680af32b8f2)

**Reply by MultimediaLucario** [reply]
Oh, thank you for bringing this to my attention.

**Reply by MultimediaLucario** [reply]
I will fix this ASAP.

**Reply by MultimediaLucario** [reply]
I just fixed it and I will put out the patch in a few moments.

**Reply by MultimediaLucario** [reply]
https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/releases/tag/v1.5.1

**Reply by VinylNerd** [reply]
Amazing, can you now try bootarg "unfairgva=4" and do a benchmark with and without this arg, perf should go up 5-20%

**Reply by MultimediaLucario** [reply]
I just tried it and I got to say it was in the 3000s range rather than 2000s. Thank you for teaching me this trick. Another release is already on the horizon.

**Reply by VinylNerd** [reply]
No problem :) i al glad you found the same. I use my laptop for livestreaming and i noticed as soon as i had it off that i started getting lagging issues and had to turn my settings down so it seems to make a large difference, i found this setting in another T480 build and was confused why they had it. Im have a bit more stability/testing to go on my build but should have EFI end of this month, i think bios unlock with undervolt will improve perf alot based on a reddit post i saw someone hitting 3900-4k multicore

**Reply by MultimediaLucario** [reply]
Which one of those IDs do I use for an i5-7300U?

**Reply by VinylNerd** [reply]
I would use same or 1659

**Reply by MultimediaLucario** [reply]
Thanks. I really do appreciate it.
